src/index.ts

```diff
@@ -1323,52 +1323,47 @@ export class MintSDK {
    * payment intent data will be shown
    *
    * Parameters:
-   * contractAddress: address where the contract is deployed
-   * networkId: network of the contract
-   * methodName: the name of method to be invoked
-   * abi: the contract interface
-   * contractMethodArgs: argument that will be passed to the `methodName` method
-   *
+   * paymentIntentId: payment intent id
    *
    * ```typescript
    * import { MintSDK } from '@kyuzan/mint-sdk-js'
    *
    * const sdk = new MintSDK(...)
-   * await sdk.requestPaymentWithPaymentIntent({
-   *  contractAddress: '0xxxxx',
-   *  networkId: 1,
-   *  methodName: 'confirmPayment',
-   *  abi: '[{"type":"function","inputs":[{"type":"string","name":"symbol"},...],...},...]',
-   *  contractMethodArgs: ["someAddress", "10", "ETH",...]
-   * })
+   * await sdk.requestPaymentWithPaymentIntentId(assdfUD1F234sdf1)
    * ```
    */
-  public requestPaymentWithPaymentIntent = async (arg: {
-    contractAddress: string
-    networkId: NetworkId
-    methodName: string // confirmPayment from the contract
-    abi: string // Application Binary Interface -
-    contractMethodArgs: any[]
-  }) => {
+  public requestPaymentWithPaymentIntent = async (paymentIntentId: string) => {
```

Comment:
Can we use overloads for backwards compatibility? 

https://dmitripavlutin.com/typescript-function-overloading/#:~:text=Function%20overloading%20in%20TypeScript%20lets,the%20function%20should%20be%20invoked.